docs/source/learn/eager.md

```diff
@@ -60,31 +76,33 @@ input = torch.randn(64, 3, 224, 224).to(device)
 res = compiled_model(input)
 ```
 
-Note that
+where the implementation of `torch_xla.compile` can be summarized as follow:
 
-1.  Currently user has to manually enable the eager mode by
-    `torch_xla.experimental.eager_mode(True)`.
-2.  The region of the code that wants to be compiled should be wrapped
-    by `torch_xla.compile`.
+1. **Disables Eager Mode**: Temporarily switches to tracing to build a
+    computation graph.
 
-The implementation of the `torch_xla.compile` is actually pretty
-straight forward, it disables the eager mode when entering the target
-function and start tracing. It will call the `torch_xla.sync()` when
-target function returns and reenable the eager mode. You can expect the
-same perfomrance by using the `eager` + `compile` API compared to the
-existing `mark_step/sync` approach.
+2. **Traces Operations**: Records operations for XLA optimization.
 
-### Inference
+3. **Compiles and Executes**: Triggers compilation and execution via an
+    internal `torch_xla.sync()` call.
 
-``` python
-torch_xla.experimental.eager_mode(True)
-compiled_model = torch.compile(model, backend="openxla")
-```
+4. **Re-enables Eager Mode**: Resumes eager execution after compilation.
+
+This "eager-to-lazy-to-eager" transition abstracts synchronization complexity,
+balancing flexibility and performance.
 
-It is recommened to use the `torch.compile` instead of
-`torch_xla.compile` for inference to reduce the tracing overhad.
+## `torch_xla.compile` vs. `torch.compile`
 
-### Training
+The PyTorch ecosystem offers multiple compilation APIs, and understanding their
+distinct roles, especially within PyTorch/XLA, is crucial for optimal
+performance and development.
+
+- `torch_xla.compile` is optimized for PyTorch/XLA training workflows.  Designed
+to work efficiently with the XLA backend for iterative training, it's the
+recommended API for compiling training loops due to its observed performance
+advantages. Best practice dictates enclosing the complete training step—forward
```

Comment:
The best practice is to enclose the complete training step-forward pass, ...

Reply:
PTAL.